src/Components/IndexScene/IndexScene.tsx

```diff
@@ -496,40 +497,51 @@ export class IndexScene extends SceneObjectBase<IndexSceneState> {
   private limitMaxInterval(timeRange: SceneTimeRangeLike) {
     return (newState: SceneTimeRangeState, prevState: SceneTimeRangeState) => {
       const { jsonData } = plugin.meta as AppPluginMeta<JsonData>;
-      if (jsonData?.interval) {
```

Comment:
Nit, but I would appreciate if the indentation level in this function is improved. Can return early a bunch of times.